sample/sample.go

```diff
@@ -113,7 +116,53 @@ func makeSampleFromZIP(src, outDir string, m int) error {
 	return nil
 }
 
-func makeSample(src, outDir string, m int) error {
+func makeUpdatedAtSample(src, outDir string, updatedAt string, mockUpdatedAt bool) error {
+	copyFile := func(src io.Reader, target, name string) error {
+		w, err := os.Create(target)
+		if err != nil {
+			return fmt.Errorf("error creating %s: %w", target, err)
+		}
+		defer w.Close()
+
+		if _, err := io.Copy(w, src); err != nil {
+			return fmt.Errorf("error copying %s to sample directory: %w", name, err)
+		}
+
+		return nil
+	}
+
+	name := filepath.Base(src)
+	out := filepath.Join(outDir, name)
+
+	r, err := os.Open(src)
+	if err != nil {
+		if mockUpdatedAt {
```

Comment:
Acho que aqui faz sentido verificar qual erro, não?

```suggestion
	if err != nil {
		if os.IsNotExist(err) {
			if mockUpdatedAt {
```